.github/workflows/test.yml

```diff
@@ -32,6 +32,8 @@ jobs:
           key: pnpm-${{ hashFiles('pnpm-lock.yaml') }}
       - name: Install dependencies
         run: pnpm install --frozen-lockfile --ignore-scripts
+      - name: Run projects postinstall
+        run: pnpm -r postinstall
```

Comment:
Will it run all `postinstall` scripts in all dependencies?

What if we will do more explicit thing:

1. We will add `build:countries` script
2. Call this script in `postinstall` script
3. In CI call `build:countries` explicitly